 public Throwable getCause() { return causeException; } 